changelog/14869.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:change
+auth: Remove support for legacy MFA
+(https://www.vaultproject.io/docs/auth/mfa)
```

Comment:
This link will likely not exist any longer given that the doc page has been removed, correct? Should we link to a specific version of the docs like in the deprecation table?

Reply:
For sure. Thanks for catching that!